src/rust/engine/rule_graph/src/lib.rs

```diff
@@ -435,19 +435,12 @@ impl<'t, R: Rule> GraphMaker<'t, R> {
         // If no candidates were fulfillable, this rule is not fulfillable.
         unfulfillable_diagnostics.push(Diagnostic {
           params: params.clone(),
-          reason: if params.is_empty() {
-            format!(
-              "No rule was available to compute {}. Maybe declare RootRule({})?",
-              dependency_key, product,
-            )
-          } else {
-            format!(
-              "No rule was available to compute {} with parameter type{} {}",
-              dependency_key,
-              if params.len() > 1 { "s" } else { "" },
-              params_str(&params),
-            )
-          },
+          // TODO(#9621)
```

Comment:
I usually prefer to leave a few words describing the relationship of the TODO ticket to the code I'm commenting -- but not necessary if the ticket contains sufficient context.

Reply:
Oh, oops, I totally meant to fill this in. I forgot to. Thanks!